devices/xiaomi.js

```diff
@@ -1301,7 +1301,10 @@ module.exports = [
         exposes: [e.power().withAccess(ea.STATE_GET), e.energy(), e.temperature(), e.voltage().withAccess(ea.STATE), e.current(),
             e.switch().withEndpoint('l1'), e.switch().withEndpoint('l2'),
             exposes.binary('interlock', ea.STATE_SET, true, false)
-                .withDescription('Enabling prevents both relais being on at the same time')],
+                .withDescription('Enabling prevents both relais being on at the same time'),
+            exposes.numeric('power_outage_count', ea.STATE)
+                .withDescription('Number of power outages (since last pairing)'),
```

Comment:
Another thing I've observed is that the number of outages does not return to zero when pairing for LLKZMK11LM. It returns to zero for SSM-U01. So I think I will change this message.

Reply:
Message changed.

@Koenkk maybe is better to add a new preset for this, like this:

https://github.com/Koenkk/zigbee-herdsman-converters/blob/bf49c91feff85fb8812b0b59a87813949cc9bebd/lib/exposes.js#L585

With a text that does not specify if it returns to zero or not when pairing or with a parameter that changes the text? There are a more devices that have this attribute.

Reply:
Added new commit with the preset.